Images/ImageImportExport/ImageExport.cpp

```diff
@@ -23,181 +18,112 @@
 #include "PagePDECntCalls.h"
 #include "PEWCalls.h"
 
-#define BTS(b) ((b) ? "true" : "false") // Boolean to String
-
 // This variable is used to give exported files unique names for the purposes of this sample.
 // The values is incremented every time a PDEImage is found in the PDF document.
-int nextImg = 1;
+CosID nextImg = 1;
```

Comment:
What I mean here was really querying the image for its CosID but then again thinking about it again, I don't want to overcomplicate the sample, we can just keep it as an integer.